<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlipRotateTool</class>
 <widget class="QWidget" name="FlipRotateTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="activeHLayout">
    <item>
     <widget class="QLabel" name="activeLabel">
      <property name="text">
       <string>Active:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="activeComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="flipGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>31</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Flip</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>22</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="flipVLayout">
     <item>
      <widget class="QPushButton" name="flipAxialButton">
       <property name="toolTip">
        <string>Flip across the axial plain</string>
       </property>
       <property name="text">
        <string>Axial</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="flipCoronalButton">
       <property name="toolTip">
        <string>Flip across the coronal plain</string>
       </property>
       <property name="text">
        <string>Coronal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="flipSagittalButton">
       <property name="toolTip">
        <string>Flip across the sagittal plain</string>
       </property>
       <property name="text">
        <string>Sagittal</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="rotateGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Rotate</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>22</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="rotateVLayout">
     <item>
      <widget class="QPushButton" name="rotateACButton">
       <property name="toolTip">
        <string>Rotate axial to coronal</string>
       </property>
       <property name="text">
        <string>Axial - Coronal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rotateASButton">
       <property name="toolTip">
        <string>Rotate axial to sagittal</string>
       </property>
       <property name="text">
        <string>Axial - Sagittal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rotateSAButton">
       <property name="toolTip">
        <string>Rotate sagittal to axial</string>
       </property>
       <property name="text">
        <string>Sagittal - Axial</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
